### PR TITLE
Fix: Select class modal now shows base items

### PR DESCRIPTION
**Pull request by CuriousMagpie**
Changes:
- Each class avatar now shows the class's best "basic" gear instead of Gala gear
- Avatars have been slightly nudged so they're all more or less centered and no longer move when you select one
- Updated CSS selectors to `px` from `em` where appropriate (margins, padding)

### CODE REVIEW

File: website/client/src/components/achievements/chooseClass.vue [hunk 1]
```diff
@@ -121,44 +121,48 @@
...
+    font-size: 1.5rem;
     font-weight: bold;
-    margin: auto 0.33333em;
+    margin: auto 5px;
   }
 
   .danger {
     color: $red-50;
-    margin-bottom: 0em;
+    margin-bottom: 0px;
```
Comment (Tressley) [inline website/client/src/components/achievements/chooseClass.vue]: We don't need to use units when the value is 0.

Comment (CuriousMagpie) [reply website/client/src/components/achievements/chooseClass.vue]: Updated!

File: website/client/src/components/achievements/chooseClass.vue [hunk 2]
```diff
@@ -121,44 +121,48 @@
...
-    margin-bottom: 0em;
+    margin-bottom: 0;
   }
 
   .header-purple {
     color: $purple-200;
-    margin-top: 1.33333em;
-    margin-bottom: 0em;
+    margin-top: 40px;
+    margin-bottom: 0px;
```
Comment (Tressley) [inline website/client/src/components/achievements/chooseClass.vue]: Got a straggler here...

Comment (CuriousMagpie) [reply website/client/src/components/achievements/chooseClass.vue]: Updated!

File: website/client/src/components/achievements/chooseClass.vue [hunk 3]
```diff
@@ -121,44 +121,48 @@
   }
 
   .class-explanation {
-    font-size: 16px;
-    margin: 1.5em auto;
+    font-size: 1rem;
```
Comment (SabreCat) [inline website/client/src/components/achievements/chooseClass.vue:124]: Are we changing to use `rem` for font-size?

File: website/client/src/components/achievements/chooseClass.vue [hunk 4]
```diff
@@ -121,44 +121,48 @@
...
+    font-size: 1.5rem;
     font-weight: bold;
-    margin: auto 0.33333em;
+    margin: auto 5px;
   }
 
   .danger {
     color: $red-50;
-    margin-bottom: 0em;
+    margin-bottom: 0;
```
Comment (SabreCat) [inline website/client/src/components/achievements/chooseClass.vue]: Anywhere you're setting a `margin-bottom: 0;` you can use the `mb-0` helper class in the html, will save you some CSS lines

**Review by SabreCat — CHANGES_REQUESTED** [review]
One question, one suggestion, easy to resolve!

File: website/client/src/components/achievements/chooseClass.vue (hunk 3 above)
Comment (CuriousMagpie) [reply website/client/src/components/achievements/chooseClass.vue:124]: We aren't. Updated to `em`

File: website/client/src/components/achievements/chooseClass.vue (hunk 4 above)
Comment (CuriousMagpie) [reply website/client/src/components/achievements/chooseClass.vue]: Updated!

File: website/client/src/components/achievements/chooseClass.vue (hunk 3 above)
Comment (Tressley) [reply website/client/src/components/achievements/chooseClass.vue:124]: I would prefer we use `rem` over `em`, but it should be `px` to be consistent?

Comment (SabreCat) [reply website/client/src/components/achievements/chooseClass.vue:124]: oh I see! If `rem` is good let's do it!
Definitely not `em`, where did you get that idea 😵‍💫

Comment (CuriousMagpie) [reply website/client/src/components/achievements/chooseClass.vue:124]: Back to `rem` we go! 🫠

**Review by SabreCat — APPROVED** [review]

**Pull request merged**